File: app/src/test/java/com/zl/weilu/androidut/mockito/MockitoRuleTest.java
package com.zl.weilu.androidut.mockito;

import com.zl.weilu.androidut.bean.Person;

import org.junit.Rule;
import org.mockito.Mock;
import org.mockito.junit.MockitoJUnit;
import org.mockito.junit.MockitoRule;

/**
 * @Description: MockitoRule方式Mock
 * @Author: weilu
 * @Time: 2017/11/4 14:43.
 */

public class MockitoRuleTest {

    @Mock 
    Person mPerson;

    @Rule
    public MockitoRule mockitoRule = MockitoJUnit.rule();
    
    
   
}

File: app/src/test/java/com/zl/weilu/androidut/mockito/MockitoRuleTest.kt
package com.zl.weilu.androidut.mockito

import com.zl.weilu.androidut.bean.Person
import org.junit.Assert.assertNotNull
import org.junit.Rule
import org.junit.Test
import org.mockito.Mock
import org.mockito.junit.MockitoJUnit

/**
 * @Description: MockitoRule方式Mock
 * @Author: weilu
 * @Time: 2017/11/4 14:43.
 */

class MockitoRuleTest {

    @Mock
    lateinit var mPerson: Person

    @get:Rule
    var mockitoRule = MockitoJUnit.rule()

    @Test
    fun testIsNotNull() {
        assertNotNull(mPerson)
    }

}
